src/date-picker/classes.scss

```diff
@@ -5,3 +5,15 @@
 .iui-date-picker {
   @include iui-date-picker;
 }
+
+.iui-calendar-month-year {
+  @include iui-calendar-month-year;
+}
+
+.iui-calender-weekdays {
```

Comment:
```suggestion
.iui-calendar-weekdays {
```

Reply:
oh wow, nice catch...changed.
Why my spell checker didnt catch this one :/